onyo/lib/commands.py

```diff
@@ -116,6 +116,8 @@ def onyo_cat(inventory: Inventory,
         YAML format.
     """
     from .assets import validate_yaml
+    if not paths:
+        raise ValueError("At least one asset path must be specified.")
```

Comment:
Agree on the addition, but maybe add it in the docstring for when a `ValueError` is raised?

Reply:
Good point, I added it here and everywhere else were I added such a check!